test/BUILD.gn

```diff
@@ -621,6 +621,10 @@ if (!is_android) {
         "//testing/gmock",
         "//testing/gtest",
       ]
+
+      if (enable_brave_vpn) {
```

Comment:
I don't think this flag makes sense here for general chrome/installer/setup unit tests.

Reply:
Updated to have a VPN specific unit test target in https://github.com/brave/brave-core/pull/20754/commits/a533769f71fd425aae4a05b94ce4245a1daada90